info/Community/community-tools.md

```diff
@@ -1,29 +1,30 @@
 ---
 Description: Fun tools for omg.lol, made by our amazing community  
-Last Updated: 2022-10-08 01:52 AM EDT
+Last Updated: 2024-02-15 17:52 PM EDT
 ---
 
 # Community Tools
 
 The omg.lol community makes really cool stuff. Here’s a list of some great things made by our awesome members for omg.lol.
 
-## Profile tools
+## Now page things
 
-- [Profiler](https://profiler.omg.lol) - An easy way to create your own custom themes.
-- [OMGEX](https://github.com/LITDevs/OMGex) - Extended profile features for omg.lol.
+- [omgnow.js](https://omgnow.rknight.me/), for showing your omg.lol now page on any site you want! by [Robb Knight](https://robb.omg.lol).
+- If you want to add your last played song on Last.fm to your profile, [Robb Knight](https://robb.omg.lol)'s got you covered with [].
```

Comment:
You forgot to link to this! [recentfm.js](https://recentfm.rknight.me)

Reply:
shit